frontend/src/views/ShootList.vue

```diff
@@ -367,11 +363,81 @@ export default {
+        { text: 'NAME', value: 'name', class: 'nowrap', align: 'left', checked: false, defaultChecked: true, hidden: false },
+        { text: 'INFRASTRUCTURE', value: 'infrastructure', class: 'nowrap', align: 'left', checked: false, defaultChecked: true, hidden: false },
+        { text: 'SEED', value: 'seed', align: 'left', class: 'nowrap', checked: false, defaultChecked: false, hidden: false },
+        { text: 'TECHNICAL ID', value: 'technicalId', class: 'nowrap', align: 'left', checked: false, defaultChecked: false, hidden: !this.isAdmin },
+        { text: 'CREATED BY', value: 'createdBy', class: 'nowrap', align: 'left', checked: false, defaultChecked: false, hidden: false },
+        { text: 'CREATED AT', value: 'createdAt', class: 'nowrap', align: 'left', checked: false, defaultChecked: false, hidden: false },
+        { text: 'PURPOSE', value: 'purpose', class: 'nowrap text-center', align: 'center', checked: false, defaultChecked: true, hidden: false },
+        { text: 'STATUS', value: 'lastOperation', class: 'nowrap text-left', align: 'left', checked: false, defaultChecked: true, hidden: false },
+        { text: 'VERSION', value: 'k8sVersion', class: 'nowrap text-center', align: 'center', checked: false, defaultChecked: true, hidden: false },
+        { text: 'READINESS', value: 'readiness', class: 'nowrap text-center', sortable: true, align: 'center', checked: false, defaultChecked: true, hidden: false },
+        { text: 'ACCESS RESTRICTIONS', value: 'accessRestrictions', sortable: false, align: 'left', checked: false, defaultChecked: false, hidden: !this.cfg.accessRestriction || !this.isAdmin },
+        { text: 'TICKET', value: 'ticket', class: 'nowrap', sortable: true, align: 'left', checked: false, defaultChecked: false, hidden: !this.gitHubRepoUrl || !this.isAdmin },
+        { text: 'TICKET LABELS', value: 'ticketLabels', sortable: false, align: 'left', checked: false, defaultChecked: true, hidden: !this.gitHubRepoUrl || !this.isAdmin },
+        { text: 'ACTIONS', value: 'actions', class: 'nowrap text-right action-button-group', sortable: false, align: 'right', checked: false, defaultChecked: true, hidden: !(this.canDeleteShoots || this.canGetSecrets) }
+      ]
+      return map(headers, (header, index) => ({
+        ...header,
+        weight: (index + 1) * 100,
+        checked: get(this.checkedColumns, header.value, header.defaultChecked)
+      }))
+    },
+    customHeaders () {
+      const customHeaders = filter(this.customFieldsListShoot, ['showColumn', true])
+
+      return map(customHeaders, ({
+        align = 'left',
+        name,
+        key,
+        path,
+        columnSelectedByDefault: defaultChecked,
+        tooltip,
+        defaultValue,
+        sortable,
+        weight
+      }, index) => {
+        return {
+          customField: true,
+          text: upperCase(name),
+          value: key,
```

Comment:
Why is this named value if it is the key or name of the custom field?

Reply:
`value` is defined / required by vuetify for a header